package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "react-dropdown-now",
-  "version": "1.0.9",
-  "description": "React dropdown component NOW",
+  "version": "1.1.0",
```

revert this line. will be auto incremented by release command

```suggestion
  "version": "1.0.9",
```